nuxeo-common/src/test/java/org/nuxeo/common/codec/CryptoTest.java

```diff
@@ -98,18 +98,22 @@ public void testDecryptionFailSilently() throws Exception {
         assertFalse(Crypto.isEncrypted(""));
         assertFalse(Crypto.isEncrypted("{$$}"));
 
+        // bad padding
         String encryptedString = "{$$xoWODkBtRXnsdzIBdosUFA==}";
         String decryptedString = new String(crypto.decrypt(encryptedString));
         assertEquals("Decryption must fail silently", decryptedString, encryptedString);
 
+        // last block FB== is not valid (B is not a possible value)
         encryptedString = "{$$xoWODkCtRXnSuqIBdosUFB==}";
         decryptedString = new String(crypto.decrypt(encryptedString));
         assertEquals("Decryption must fail silently", decryptedString, encryptedString);
 
+        // illegal block size
         encryptedString = "{$REVTL0VDQi9QS0NTNVBhZGRpbmc=$w/wCDLXoGsasskNaEpMt0wg==}";
         decryptedString = new String(crypto.decrypt(encryptedString));
         assertEquals("Decryption must fail silently", decryptedString, encryptedString);
 
+        // bad padding
         encryptedString = "{$REVTL0VDQi9QS0NTNVBhZGRpbmc=$w/wCDLXoG5VFkNbEpMt0wg==}";
         decryptedString = new String(crypto.decrypt(encryptedString));
         assertEquals("Decryption must fail silently", decryptedString, encryptedString);
```

Comment:
Why doing several bad padding cases ? Maybe the answer should be in the comments ?
How about extracting the error message in a local variable ?

Reply:
These are not my tests, I just added a small bit of doc about the error cases while resolving a related issue.
Also this is a test, putting common error messages in variables or constants doesn't bring anything. In tests it's ok to repeat things sometimes.